react-app/src/components/map/ParcelMap.tsx

```diff
@@ -76,6 +77,15 @@ const ParcelMap = (props: ParcelMapProps) => {
     api.properties.propertiesGeoSearch(filter),
   );
 
+  const deletionBroadcastChannel = useMemo(() => new BroadcastChannel('property'), []);
+  useEffect(() =>
+    deletionBroadcastChannel.addEventListener('message', (event) => {
+      if (typeof event.data === 'string' && event.data === 'refresh') {
+        refreshData();
+      }
+    }),
```

Comment:
If we were to store the map view before the refresh, then reapply it after, maybe that would avoid the zooming out issue?

Reply:
Good suggestion but changes to accommodate this would be potentially breaking, so we will come back to this later.